builtin/k8s/platform.go

```diff
@@ -578,6 +607,15 @@ deploy "kubernetes" {
 		),
 	)
 
+	doc.SetField(
+		"ports",
+		"a map of ports and options that the application is listening on",
+		docs.Summary(
+			"used to define and configure multiple ports that the application is",
+			"listening on. Can also define a 'name', 'host_port', and 'host_ip'.",
```

Comment:
I think this would be more clear if we should all of the map keys as separate doc fields, like
```
doc.SetField("ports.port, "the port")
doc.SetField("ports.name", "name of the port")
doc.SetField("ports.host_port", "how we use the host port")
```
etc

Reply:
Or maybe just add `port` to your list for clarity, so like
```suggestion
			"listening on. Available keys are 'port', 'name', 'host_port', and 'host_ip'.",
```
(Also update wording on releaser doc field)

Reply:
Yeah, maybe we can document each option `ports` will support separately like the first comment suggests just so it's clear what each one does in the docs.

Reply:
I'm actually not sure if `doc.SetField` can create sub docs like `ports.port` and `ports.name`. They might end up as new entries? If that's the case, I'd probably want to keep it all together and just define them inside the summary strings.